test/specs/EventsSpec.js

```diff
@@ -22,96 +22,92 @@ describe("Event Tests", function() {
 
     describe("Mouse Events", function() {
 
-      it("'slideStart' event is triggered properly and can be binded to", function() {
+      it("'slideStart' event is triggered properly and can be binded to", function(done) {
         testSlider.on('slideStart', function() {
-          flag = true;
+          done();
         });
         testSlider.data('slider')._mousedown(mouse);
-        expect(flag).toBeTruthy();
       });
 
-      it("'slide' event is triggered properly and can be binded to", function() {
+      it("'slide' event is triggered properly and can be binded to", function(done) {
         testSlider.on('slide', function() {
-          flag = true;
+          done();
         });
         testSlider.data('slider')._mousemove(mouse);
-        expect(flag).toBeTruthy();
       });
 
-      it("'slide' event sets the right value on the input", function() {
+      it("'slide' event sets the right value on the input", function(done) {
         testSlider.on('slide', function() {
-          flag = true;
           expect(isNaN(testSlider.val())).not.toBeTruthy();
+          done();
         });
         testSlider.data('slider')._mousemove(mouse);
-        expect(flag).toBeTruthy();
       });
 
-      it("'slide' event value and input value properties are synchronous", function() {
+      it("'slide' event value and input value properties are synchronous", function(done) {
         testSlider.on('slide', function(e) {
-          flag = true;
           expect(e.value.toString()).toEqual(this.value);
+          done();
         });
         testSlider.slider("setValue", 3, true, false);
-
-        expect(flag).toBeTruthy();
       });
 
-      it("'change' event value and input value properties are synchronous", function() {
+      it("'change' event value and input value properties are synchronous", function(done) {
```

Comment:
No expectations... you get the point.

Reply:
https://github.com/seiyria/bootstrap-slider/pull/902/files#r243743659